SECURITY.rst

```diff
@@ -0,0 +1,124 @@
+
+**As long as you comply with this policy:**
+
+* We consider your security research to be "authorized" under the Computer Fraud and Abuse Act (and/or similar state laws), and
+* We waive any restrictions in our application Terms of Use and Usage Policies that would prohibit your participation in this policy, but only for the limited purpose of your security research under this policy.
+
+Notification Requirement
+========================
+
+* Safe harbor under this policy is only extended if the discoverer of the vulnerability notifies Astronomer as outlined elsewhere in this policy, prior to notifying any other third-party entities, and does not notify any other third-party entities for 90 days after notifying Astronomer, without Astronomer’s prior written approval.
+* After notification of Astronomer and the lapse of the 90 day period, it is requested that any publications, third-party releases, or other disseminations of information related to or derived from the vulnerability discovery be coordinated with Astronomer prior.
+
+Right to rescind safe harbor protections
+========================================
+
+Astronomer reserves the right to rescind any and all safe harbor protections originally extended to the vulnerability
+discoverer in the event that the discoverer, at any point prior to or after notification to Astronomer,
+has knowingly and willfully released, published, or otherwise used information related to the discovered vulnerability in a manner that:
+
+1. Maligns or damages the reputation of Astronomer, its customers, or its employees;
+2. Is used to conduct malicious attacks against Astronomer systems, regardless of whether material damages occur; or
+3. Exacerbates existing vulnerabilities or threats, thereby increasing the risk to Astronomer or its stakeholders.
+
+Extension of safe harbor to third-party systems and services
+============================================================
+
+Astronomer systems and services may be interconnected with third-party systems and services.
+If you submit a report through our vulnerability disclosure program that affects a third-party service,
+we will limit what we share with the affected third party.
+Please understand that, while we can authorize your research on Astronomer’s systems and services,
+we cannot authorize your efforts on third-party products or guarantee they won’t pursue legal action against you.
+That said, if legal action is initiated by a third party against you because of your participation in this vulnerability
+disclosure program, and you have complied with our vulnerability disclosure policy, we will take steps to make it known
+that your actions were conducted in compliance with this policy.
+This is not, and should not be understood as, any agreement on our part to defend, indemnify, or otherwise protect you
+from any third-party action based on your actions.
+
+You are expected, as always, to comply with all applicable laws.
```

Comment:
```suggestion
Astronomer systems and services can interconnect with third-party systems and services.
If you submit a report that affects a third-party service through the `vulnerability disclosure program <https://www.astronomer.io/vulnerability-disclosure/>`_,
Astronomer will limit what we share with the affected third party.
Please understand that, while we can authorize your research on Astronomer’s systems and services,
we cannot authorize your efforts on third-party products or guarantee they won’t pursue legal action against you.
If legal action is initiated by a third party against you because of your participation in this vulnerability
disclosure program, and you have complied with our vulnerability disclosure policy, we will take steps to make it known
that your actions were conducted in compliance with this policy.
This is not, and should not be understood as, any agreement on Astronomer's part to defend, indemnify, or otherwise protect you
from any third-party action based on your actions.

You are expected, as always, to comply with all applicable laws.
```

Reply:
I can never figure out the linebreaks with .rst - please adjust the spacing as you see fit!